domains/note-lrmn.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "lrmn7",
+        "email": "lrmnid@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "github.com/lrmn7/blogs"
```

Comment:
```suggestion
        "URL": "https://github.com/lrmn7/blogs"
```

Reply:
The website has been unpublished, please delete it